### DISCUSSION TITLE
如何減少特定文字的出現頻率？

### DISCUSSION THREAD

**Discussion by LiyJessica** [rime/home#1211] [Q&A]
我正在使用用字為香港傳統漢字(t2hkf.json)的小鶴雙拼，當我打uo的時候「説」和「說」總會同時出現，而且「説」排在「說」前面，我希望把「説」這個異體字的出現順序弄到後面一點，免得我總是打錯，請問該如何設定？
![image](https://user-images.githubusercontent.com/44195673/218477865-abe5962d-4c0b-4513-9ae4-a386d018f482.png)

**Comment by LEOYoon-Tsaw** [comment]
因為按香港標準就説，想改的話要去改OpenCC，找到用戶文件夾/opencc/t2hk.txt
如果是ocd的話得先用opencc_dict反編譯再改
